Changes.md

```diff
@@ -20,6 +20,7 @@ Improvements
 - Arnold : Added metadata for new `standard_volume` shader parameters introduced in Arnold 7.1.3.0.
 - VectorDataWidget : Added a color swatch column for `Color3f` and `Color4f` elements. These are currently included in the `RandomChoice` node when `choices` is set to a list of colors, and for color primitive variables in the Primitive Inspector.
 - NameWidget : Added a check to prevent users from setting a node or plug name to a value starting with a double underscore.
+- Light Editor : Don't show `Show History...` popup menu for `Name` column.
```

Comment:
The original feature hasn't been released yet, so we don't need to mention the improvements - it'll just confuse people.

Reply:
Ah right, fixed in a52bdd66121d8439f0c8f1a9d2cd2d9ec34c9109